tests/components/storyshot.test.ts

```diff
@@ -0,0 +1,15 @@
+import initStoryshots from '@storybook/addon-storyshots';
```

Comment:
it seems a little odd for the `storyshot.test.ts` file to live in the components directory because it doesn't match the structure of our `src` folder and test a single component. maybe it could be moved a level up?